app/Helpers/Helpers.php

```diff
@@ -163,7 +168,7 @@ function cacheRememberByKey(string $key, Carbon $ttl, Closure $callback)
 
     // cache not set. try to acquire lock to gain access to the callback
     $lock = createCacheLock($key);
-    if ($lock->acquire()) {
+    if ($lock->get()) {
```

Comment:
The get method invokes acquire method but I saw that in the Laravel documentation they suggest to use `get` rather `acquire` to know if the lock object is acquire currently.

You can see the laravel source code related with the Lock class.
https://github.com/laravel/framework/blob/9.x/src/Illuminate/Cache/Lock.php#L88